lib/rucio/web/rest/flaskapi/v1/replica.py

```diff
@@ -272,6 +274,7 @@ def delete(self):
 
 class ListReplicas(MethodView):
 
+    @check_accept_header_wrapper_flask(['application/json', 'application/metalink4+xml'])
```

Comment:
Why `application/json` here and not `application/x-json-stream` as defined below?
It's a bit strange anyway, since we define `application/x-json-stream` but the data in line 370 is not yielded...

Reply:
Ah yes. Because its for flask and we dont have streaming there at the moment. Should I change to `application/x-json-stream` as we are changing it in the future anyway?

Reply:
Ah understood.
I would say we use the ones from weby 1:1, even if flask does not allow it. Since we are not using flask anyway (other than the doc generation)